components/automate-ui/src/app/modules/team/team-management/team-management.component.html

```diff
@@ -52,10 +43,9 @@
                       <span>Name</span>
```

Comment:
Without exception, all other uses do **not** wrap these in `<span>` elements so should now be able to just do:
```
<chef-table-header-cell class="name-column">Name</chef-table-header-cell>
```

(And similarly for L46, L49, and L60.)